docs/CONTRIBUTING.md

```diff
@@ -19,126 +19,6 @@ Before creating an issue:
 
 Please submit pull requests in response to open issues. If you have a bug or feature, bring it up in issues first, to avoid two or more people working on the same thing. See workflow information below.
 
```

Comment:
Can you add a section here on setting up the environment, but let the reader know that it's available in a separate doc, with a link?  That way people who come to this file looking for that info don't get lost.

Reply:
I just pushed this fix, can you review it again? Thanks :)

Reply:
Perfect.